code/modules/client/preferences_savefile.dm

```diff
@@ -19,6 +19,9 @@
 					fdel(delpath)
 				break
 		return FALSE
+    
+  if(savefile_version < 26)
+		S["key_bindings"] << deepCopyList(GLOB.keybinding_list_by_key)
```

Comment:
Is this properly indented? Travis is complaining:
`code/modules/client/preferences_savefile.dm:23:warning: if: if statement has no effect`

Reply:
@Rohesie fixed, it was some weird indentation from the merge conflicts